### PR TITLE
TASK: Update textarea docs to latest version

### PR DESCRIPTION
**Pull request by dlubitz**
Depends on https://github.com/neos/neos-ui/pull/3821

### CODE REVIEW

**Review by Sebobo — APPROVED** [review]
LGBR

File: Neos.Neos/Documentation/References/PropertyEditorReference.rst [hunk 1]
```diff
@@ -92,12 +92,18 @@ In case the text input should span multiple lines, a ``TextAreaEditor`` should b
...
-              rows: 7
+              minRows: 5
+              expandedRows: 10
+              maxRows: 20
 
 Options Reference:
 
-``rows`` (integer)
-	Number of lines this textarea should have; Default ``5``.
+``minRows`` (integer)
```
Comment (pKallert) [inline Neos.Neos/Documentation/References/PropertyEditorReference.rst:101]: In the other Options References, there is always a space between the options as far as I can see. Could you perhaps add the space for consistency?

Comment (dlubitz) [reply Neos.Neos/Documentation/References/PropertyEditorReference.rst:101]: Thank you. Done.

**Review by pKallert — APPROVED** [review]
Looks good!

**Pull request merged**